.github/workflows/python_tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Run Python Tests with No Flight Controller
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.11"
+
+    - name: Install dependencies
+      working-directory: ./radio
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
Why we need this?

Reply:
Just incase it cant find old packages due a pip version issue - takes like 15 seconds and prevents issues when installing requirements